test/test_libqasm.py

```diff
@@ -1,7 +1,7 @@
 import unittest
 
 from opensquirrel.default_gates import *
-from opensquirrel.parsing.libqasm.libqasm_ir_creator import LibqasmIRCreator
+from opensquirrel.parse.libqasm.libqasm_ir_creator import LibqasmIRCreator
```

Comment:
`parser` maybe better? But that would imply changing all of the others as well. I'm fine with this though.

Reply:
I agree, I have changed this throughout. I didn't want it initially because I don't want the names of the directories to be identical to the modules that are in it. So I ended up changing the names of some of the modules (i.e. `merger.py` and `replacer.py`)

Reply:
Nice!